### PR TITLE
[test] Add descriptions of resnet50V1 and resnet50V2 in README.md files

### PR DESCRIPTION
**Pull request by shenx3x**
@BruceDai 
Thanks

### CODE REVIEW

**Comment by BruceDai** [comment]
LGTM, thanks
@ibelem Please help merge this PR, thanks

File: test/tools/perLayerAnalyzer/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ This is an automation tool kit to automatically test the every layer of realmode
 ```
    You need modify these eight fields for the different platforms:
    + **_urlServer_**: `{string}`, Server IP address, port number 8080.
-   + **_modelName_**: `{array}`, We support **squeezenet1.1**, **mobilenetv2-1.0** two models, you need choose **["squeezenet1.1"]** , or **["mobilenetv2-1.0"]** , or **["squeezenet1.1", "mobilenetv2-1.0"]** , or **["mobilenetv2-1.0", "squeezenet1.1"]** .
+   + **_modelName_**: `{array}`, We support **squeezenet1.1**, **mobilenetv2-1.0**, **resnet50v1**, **resnet50v2**  four models.
```
Comment (ibelem) [inline test/tools/perLayerAnalyzer/README.md]: Please update about this sentence.

File: test/tools/perLayerAnalyzer/README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ This is an automation tool kit to automatically test the every layer of realmode
...
    + **_platform_**: `{string}`, target platform, support **Android**, **Mac**, **Linux** and **Windows**.
    + **_chromiumPath_**: `{string}`, **Mac**/**Linux**/**Windows**: the target chromium path **Android**: the chrome or chromium path in above Prerequisites to show the final checking results.
```
Comment (ibelem) [inline test/tools/perLayerAnalyzer/README.md]: Chromium / Chrome

**Comment by ibelem** [comment]
LGTM

**Pull request merged**